tests/test_codesyncd/buffer_handler.test.js

```diff
@@ -159,6 +164,68 @@ describe("bufferHandler", () => {
         return true;
     };
 
+    const addTab = (repoPath, created_at, tabs ) => {
+        console.log(`repoPath: ${repoPath}, created_at: ${created_at}, tabs: ${tabs}`);
+        const tab_handler = new tabEventHandler(repoPath)
+        const config_utils = new ConfigUtils();
+        const repoId = config_utils.getRepoIdByPath(repoPath);
+        // Add tabs to buffer
+        tab_handler.addToBuffer(repoId, created_at, tabs);
+        const pathData = fs.readdirSync(tabsRepo);
```

Comment:
Why is there extra logic after adding TabFile to buffer?

Reply:
I believe I added to test output of what was being returned only.

Reply:
Assert logic should be part of tests. Kindly remove from here if only added for debugging